addons/waterways/river_gizmo.gd

```diff
@@ -50,23 +128,88 @@ func has_gizmo(spatial) -> bool:
 func get_handle_name(gizmo: EditorSpatialGizmo, index: int) -> String:
 	return "Handle " + String(index)
 
+# curve points 2:0   1      3:0   1   2        4:0   1   2   3
+# ------------------------------------------------------------------
+# center         0   1        0   1   2          0   1   2   3
+# in             2   4        3   5   7          4   6   8   10
+# out            3   5        4   6   8          5   7   9   11
+# left           6   8        9   11  13         12  14  16  18
+# right          7   9        10  12  14         13  15  17  19
```

Comment:
This comment is not very useful for me. I'm sure I could parse the system you have for this eventually, but it's definitely not self evident. Could you add a comment on how to read this table?